README.md

```diff
@@ -193,9 +144,76 @@ Key:
 - You can temporarily change the heap chain order if you don't need a dependency
   in order to reduce build times.
 
+
+
 ## Coding style
 
 * HOL source code
   - Spaces, no tabs
   - No unicode
   - `snake_case` (e.g. `bir_number_of_mem_splits_def`)
+
+
+
+## Branch policy
+
+### tags
+
+tags should have as many **completed features** as possible:
+ - no cheat
+ - must correctly compile
+ - self tests must succeed
+ - code should be tested
+
+Follow these instructions whenever you merge to master:
+  - `grep` for "cheat"
+  - check that the `README` is up to date (especially tool status)
+  - find a reviewer for your Pull Request
+
+### `master` branch
+
+`master` is the branch where every feature is available, but not necessarily finalized:
+  - Can cheat, but has to be avoided
+  - Code should not be be commented out
+  - **Holmake must work**
+  - bug-fixes are ok
+  - 1 review is needed in order to merge into `master`
+
+However, **no development happens on this branch**, but rather on separate
+feature branches.
+
+**In order to prevent mayhem**, define good interfaces for your code, so that
+development won't break existing code.
+
+### Feature branches
+
+Every "somewhat" working tool should be available in the `master` branch, but new
+features or any development must go on new branches prefixed with `dev_`.
+ - branch names must be short and explicit (prefer explicit over short)
+ - every feature branch should involve small developments
+ - rebase feature branches on `master` **often**, by using `git rebase` or `git merge`
+ - **merge feature branches on `master` often**: work on small features
+
+Some rules for feature branches:
+ - commits in a feature branch must compile, unless explicitly stated in commit
+   message (with the prefix `[WIP] ...` for instance)
+ - further subbranch to do implementation experiments (keep them small)
+
+If you want to violate the rules for temporary development or experiments (only
+for feature branches):
+  1. Fork
+  2. Do a good mess
+  3. Merge in feature branch after history rewrite
+
+
+
+## References
+
+* A. Lindner, R. Guanciale och R. Metere, **"TrABin : Trustworthy analyses of binaries"**, Science of Computer Programming, vol. 174, p. 72-89, 2019. [Link](https://doi.org/10.1016/j.scico.2019.01.001). (the proof-producing binary analysis framework with weakest preconditions in HOL4)
```

Comment:
"och"?

Reply:
ooops, i copied from a KTH page and didn't see this. i fixed some other places but this i missed